### ISSUE TITLE
Android app crashes just after finding public transit options

### ISSUE THREAD

**Issue by TheFighter137** [osmandapp/OsmAnd#19600]
### Description

Hello,
When I try to find any public transport route, about 1 second after it shows the results, the Android app crashes.

### Steps to reproduce

1. Open the OsmAnd Android app
2. Enter a destination (the starting point being my location)

### Actual result

The app manages to find a few route options, but 1 second later, the app closes itself (crashes).

### Expected result

After finding the route options, the app should stay open and let me choose an option and start the navigation.

### Your Environment (required)

Crash-Logs:
[log_2024_04_17_19_30_45[1].log](https://github.com/osmandapp/OsmAnd/files/15015136/log_2024_04_17_19_30_45.1.log)

```
OsmAnd Version: 4.7.9 (latest I can get)
Android/iOS version: Android 14 (last update : 12/03/2024)
Device model: Samsung Galaxy A34

```

**Comment by yuriiurshuliak** [comment]
To reproduce it, select the "Public transport" navigation profile and choose a destination starting from your current location. Crash logs are not being displayed during this process.

OsmAnd~ 4.8.0#2106m, released: 2024-04-17


https://github.com/osmandapp/OsmAnd/assets/127092082/e23e9d74-62a9-48ff-a458-93500111c10b

**Comment by Dima-1** [comment]
Will be fixed 4.7.10

**Comment by yuriiurshuliak** [comment]
The bug has been fixed. The app does not crash after searching for public transport options.

OsmAnd~ 4.8.0#2166m, released: 2024-04-24

https://github.com/osmandapp/OsmAnd/assets/127092082/d265d879-661c-4ec3-9cc0-f7e06aed685b
